contextbasedrestrictionsv1/context_based_restrictions_v1_examples_test.go

```diff
@@ -63,22 +62,23 @@ var _ = Describe(`contextbasedrestrictionsv1.ContextBasedRestrictionsV1 Examples
 
 	var shouldSkipTest = func() {
 		if !configLoaded {
-			Skip("External configuration is not available, skipping tests...")
```

Comment:
we can probably keep calling these tests instead of examples.

Reply:
yes, if external configuration file is missing, example and integration tests will be skipped, but not the unit tests.